_search-plugins/sql/sql-ppl-api.md

```diff
@@ -228,6 +228,56 @@ POST _plugins/_ppl/_explain
 
 For queries that require post-processing, the `explain` response includes a query plan in addition to the OpenSearch DSL. For those queries that don't require post processing, you can see a complete DSL.
 
+#### Sample explain request when plugins.calcite.enabled set to true
```

Comment:
Please add some explanations about the difference using calcite.

Reply:
done